src/main/components/ResponseComponent.jsx

```diff
@@ -52,8 +59,9 @@ const ResponseComponent = (props) => {
   return (
     <ResponseWrapper>
       <Header>
+        <h3>{ name || 'Untitled test' }</h3>
```

Comment:
I really like how on the Mockups Panel, it defaults to "Test #1" then "Test #2", etc.  I think that should persist over here.

Reply:
Will make that change 👌